portal/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>org.apache.tomcat</groupId>
       <artifactId>tomcat-catalina</artifactId>
-      <version>7.0.68</version>
+      <version>9.0.34</version>
```

Comment:
how is this dependency used? Would this still allow for backwards compatibility with older versions of Tomcat i.e. running cBioPortal in a tomcat 8 version? We are using tomcat 8 in production for some instance of cBioPortal. There is a plan to migrate but for now it would be nice to be backwards compatible

That said it is fine to upgrade the `webapp-runner` dependency which is what the docker container uses to run cBioPortal.

Reply:
Thanks for the hint! I actually pushed *the wrong* commit. 
I'll have to revert that one and the upgrade to servlet 4.0.1 as it also requires Tomcat 9+.